Jquery and Angular project throws `Duplicate identifier 'cssPropertySetter'.` with @types/angular 1.6.22

After the last npm release 1.6.2. projects with  @types/angular and @types/jquery fail tslint.
This is caused by angular types now including jqlite.d.ts which clashes with jquery.d.ts

The error thrown by the tslint:

```
node_modules/@types/angular/jqlite.d.ts(799,6): error TS2300: Duplicate identifier 'cssPropertySetter'.
node_modules/@types/jquery/index.d.ts(705,6): error TS2300: Duplicate identifier 'cssPropertySetter'.
node_modules/@types/jquery/index.d.ts(707,5): error TS2374: Duplicate string index signature.
```

--- a/types/angular/index.d.ts
+++ b/types/angular/index.d.ts
@@ -1,6 +1,9 @@
 // Type definitions for Angular JS 1.6
 // Project: http://angularjs.org
-// Definitions by: Diego Vilar <http://github.com/diegovilar>, Georgii Dolzhykov <http://github.com/thorn0>, Caleb St-Denis <https://github.com/calebstdenis>
+// Definitions by: Diego Vilar <http://github.com/diegovilar>
+//                 Georgii Dolzhykov <http://github.com/thorn0>
+//                 Caleb St-Denis <https://github.com/calebstdenis>
+//                 Leonard Thieu <https://github.com/leonard-thieu>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.3
 

--- a/types/angular/jqlite.d.ts
+++ b/types/angular/jqlite.d.ts
@@ -796,6 +796,7 @@ interface JQueryEventObject extends BaseJQueryEventObject, JQueryInputEventObjec
 /**
  * The interface used to specify the properties parameter in css()
  */
+// tslint:disable-next-line:class-name
 interface cssPropertySetter {
     (index: number, value?: string): string | number;
 }

--- a/types/angular/jqlite.d.ts
+++ b/types/angular/jqlite.d.ts
@@ -169,7 +169,7 @@ interface JQuery {
      * @param properties An object of property-value pairs to set.
      * @see {@link https://api.jquery.com/css/#css-properties}
      */
-    css(properties: JQueryCssProperties): JQuery;
+    css(properties: JQLiteCssProperties): JQuery;
 
     /**
      * Store arbitrary data associated with the matched elements.
@@ -799,6 +799,6 @@ interface JQueryEventObject extends BaseJQueryEventObject, JQueryInputEventObjec
 interface cssPropertySetter {
     (index: number, value?: string): string | number;
 }
-interface JQueryCssProperties {
+interface JQLiteCssProperties {
     [propertyName: string]: string | number | cssPropertySetter;
 }

--- a/types/angular/jqlite.d.ts
+++ b/types/angular/jqlite.d.ts
@@ -796,7 +796,9 @@ interface JQueryEventObject extends BaseJQueryEventObject, JQueryInputEventObjec
 /**
  * The interface used to specify the properties parameter in css()
  */
-type cssPropertySetter = (index: number, value?: string) => string | number;
+interface cssPropertySetter {
+    (index: number, value?: string): string | number;
+}
 interface JQueryCssProperties {
     [propertyName: string]: string | number | cssPropertySetter;
 }

--- a/types/jquery/v1/index.d.ts
+++ b/types/jquery/v1/index.d.ts
@@ -702,7 +702,9 @@ interface JQueryCoordinates {
 /**
  * The interface used to specify the properties parameter in css()
  */
-type cssPropertySetter = (index: number, value?: string) => string | number;
+interface cssPropertySetter {
+    (index: number, value?: string): string | number;
+}
 interface JQueryCssProperties {
     [propertyName: string]: string | number | cssPropertySetter;
 }

--- a/types/jquery/v2/index.d.ts
+++ b/types/jquery/v2/index.d.ts
@@ -702,7 +702,9 @@ interface JQueryCoordinates {
 /**
  * The interface used to specify the properties parameter in css()
  */
-type cssPropertySetter = (index: number, value?: string) => string | number;
+interface cssPropertySetter {
+    (index: number, value?: string): string | number;
+}
 interface JQueryCssProperties {
     [propertyName: string]: string | number | cssPropertySetter;
 }

